hold.go

```diff
@@ -0,0 +1,112 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/goccy/go-json"
+)
+
+// ZoneHold represents a Zone Hold rule. A rule only permits access to
```

Comment:
The comment doesn't match the expected behaviour. You may use copy found on https://developers.cloudflare.com/api/operations/zones-0-hold-post.

Reply:
A lot of these were copy pasta from a different file. I updated not, can you check if that works for you.